logstash.go

```diff
@@ -50,6 +50,15 @@ func NewHookWithFieldsAndConnAndPrefix(conn net.Conn, appName string, alwaysSent
 	return &Hook{conn: conn, appName: appName, alwaysSentFields: alwaysSentFields, hookOnlyPrefix: prefix}, nil
 }
 
+//Make a new hook which does not forward to logstash, bu simply enforces the prefix rules
```

Comment:
The comment should be in the following format:
NewFilterHook makes a new hook which does not....